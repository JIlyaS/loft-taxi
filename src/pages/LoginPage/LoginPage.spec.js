import React from 'react';
import {LoginPage} from './';
import renderer from 'react-test-renderer';

describe('LoginPage', () => {
  it('renders correctly', () => {
      const tree = renderer.create(<LoginPage navigateTo={() => {}} />).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
