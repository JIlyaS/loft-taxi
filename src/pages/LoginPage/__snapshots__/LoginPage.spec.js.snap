// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage renders correctly 1`] = `
<div
  className="login-page"
>
  <div
    className="login-page__left"
  >
    <img
      alt="Логотип"
      className="login-page__logo"
      src="logo-auth.svg"
    />
  </div>
  <div
    className="login-page__right"
  >
    <div
      className="login-form"
    >
      <form
        onSubmit={[Function]}
      >
        <h2
          className="login-form__title"
        >
          Войти
        </h2>
        <div
          className="login-form__block"
        >
          <div
            className="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-1 MuiFormControl-marginDense MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense Mui-required Mui-required"
              data-shrink={false}
              htmlFor="email"
              id="email-label"
            >
              Email
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-marginDense MuiInput-marginDense"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={true}
                className="MuiInputBase-input MuiInput-input MuiInputBase-inputMarginDense MuiInput-inputMarginDense"
                disabled={false}
                id="email"
                name="email"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="mail@mail.ru"
                required={true}
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          className="login-form__block"
        >
          <div
            className="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-1 MuiFormControl-marginDense MuiFormControl-fullWidth"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense Mui-required Mui-required"
              data-shrink={false}
              htmlFor="password"
              id="password-label"
            >
              Пароль
              <span
                aria-hidden={true}
                className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-marginDense MuiInput-marginDense"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input MuiInputBase-inputMarginDense MuiInput-inputMarginDense"
                disabled={false}
                id="password"
                name="password"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="************"
                required={true}
                type="password"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          className="login-form__link-block"
        >
          <a
            className="login-form__forget-pass"
            href="/"
          >
            Забыли пароль?
          </a>
        </div>
        <button
          className="MuiButtonBase-root MuiButton-root WithStyles(ForwardRef(Button))-root-2 MuiButton-contained login-form__login-btn MuiButton-containedPrimary MuiButton-disableElevation MuiButton-fullWidth"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
            Войти
          </span>
        </button>
        <div
          className="login-form__new-block"
        >
          <span
            className="login-form__new-text"
          >
            Новый пользователь?
          </span>
          <a
            className="login-form__reg"
            href="/"
            onClick={[Function]}
          >
            Регистрация
          </a>
        </div>
      </form>
    </div>
  </div>
</div>
`;
